stylesheet.css

```diff
@@ -464,6 +464,53 @@ body.order-by-relevance .grid-item {
   order: var(--order-relevance);
 }
 
+/* Footer */
+.footer {
+  align-items: center;
+  background: #EEE;
```

Comment:
I'd personally go with a footer that is just part of the background, like the header, any particular reason you went with this?

Reply:
No reason particularly, just mimicking the existing site in that regard.